shared/components/Schedule/index.js

```diff
@@ -1,58 +1,72 @@
+      description: 'Blurb about morning talks',
+      details: [
+        {
+          time: '9:40 - 10:20 am',
+          title: 'Computer Hardware Desktops And Notebooks And Handhelds Oh My',
+          description: 'Stu is a founder with a delivery focus. He ensures that everything we create is both great for the client, and challenging for us.', // eslint-disable-line max-len
+          speaker: {
+            name: 'Christopher Chedau',
+            url: 'http://example.com',
+            company: 'Facebook',
+          },
+        },
+        {
+          time: '9:40 - 10:20 am',
+          title: 'Computer Hardware Desktops And Notebooks And Handhelds Oh My',
+          description: 'Stu is a founder with a delivery focus. He ensures that everything we create is both great for the client, and challenging for us.', // eslint-disable-line max-len
+          speaker: {
+            name: 'Christopher Chedau',
+            url: 'http://example.com',
+            company: 'Facebook',
+          },
+        },
+      ],
+    },
+  ],
+};
+
 const Schedule = () => {
   return (
     <div className="Schedule block">
       <div className="content">
-        <h2 className="Schedule__header">Schedule</h2>
-        <ol className="Schedule__list">
-          <li>
-            <p
-              className="Schedule__item--time Schedule__item--first"
-              data-time="8:30am – 9:30am"
-            >
-              Registration
-            </p>
-          </li>
-          <li>
-            <p className="Schedule__item">3 morning talks</p>
-          </li>
-          <li>
-            <p className="Schedule__item">1 panel discussion</p>
-          </li>
-          <li>
-            <p className="Schedule__item--time" data-time="12:40pm – 1:40pm">
-              Street food lunch catered by &#160;
-              <a
-                className="Schedule__caterer"
-                href="http://www.leithsdining.co.uk/our-food/"
-                target="_blank"
-                rel="noopener"
-              >
-                Leiths
-              </a>
-            </p>
-          </li>
-          <li>
-            <p className="Schedule__item">4 lightning talks</p>
-          </li>
-          <li>
-            <p className="Schedule__item">3 afternoon talks</p>
-          </li>
-          <li>
-            <p
-              className="Schedule__item--time Schedule__item--last"
-              data-time="6:00pm – 9:00pm"
-            >
-              Join us for a drink
-            </p>
-          </li>
-        </ol>
-        <div className="Schedule__subtext">Full schedule to come soon</div>
+        <h2 className="Schedule__name">
+          {mainEventData.name}
+        </h2>
+        <h2 className="Schedule__date">{mainEventData.date}</h2>
+        <div className="Schedule__list">
+          {mainEventData.schedule.map((scheduleItem, index) => (
+            <ScheduleItem key={index} item={scheduleItem} />
+          ))}
```

Comment:
You could make the HTML more semantic (better for screen readers). If this is tricky because of the css then please don't waste any time on it. Here's an idea of the changes followed by some example HTML:

 * Whole schedule is a list
 * Ordered lists instead of unordered ones
 * Time and title inside the heading tags

```html
<ol>
  <li>
    <h3>
      <time>9.40-11:00 am</time>
      Morning Talks
    </h3>
    <ol>
      <li>
        <h4>
          <time>9.40-10:20 am</time>
          Computer Hardware Desktops…
        </h4>
        <p>Stu is a founder with a delivery…</p>
      </li>
    </ol>
  </li>
</ol>
```